moduleframework/dockerlinter.py

```diff
@@ -228,6 +228,19 @@ def check_chained_run_rest_commands(self):
             return False
         return True
 
+    def check_clean_all_command(self):
+        """
+        Function checks if Dockerfile contains dnf/yum clean all in RUN instruction,
```

Comment:
Suggestion for better docstring:

```
This function checks whether every RUN instruction containing a dnf/yum operation ends with a "dnf/yum clean all".

:return: True if every dnf/yum instruction contains a cleanup step
         False otherwise
```
```